src/content/docs/guides/sysadmin.md

```diff
@@ -81,37 +80,40 @@ sc start mapepire
 nohup /QOpenSys/pkgs/bin/mapepire &
 ```
 
-
 ### Manual shell invocation (manually-installed version)
 
 ```sh
 nohup /opt/mapepire/bin/mapepire &
 ```
 
 ----
-# Configuration
 
-## Port configuration
+## Configuration
+
+### Server Name
+
+Make sure any name reference to the server (as opposed to a dotted IP address), such as the `DB_SERVER` variable in your `.env` file, points to a name that can be found on the IBM is via DNS. You may have to create a host table entry (see the IBM i command `CFGTCP` and its `Option 10 Work with TCP/IP host table entries`)
```

Comment:
We don't actually recommend users use the `.env` with Mapepire. This is purely for dev and test only. Perhaps we should remove the check from production code (cc @ThePrez) but I don't think this should be in the documentation.

Reply:
Does then the server still need to be able to find its name via DNS? Or is that an artifact of using `.env`?

Reply:
Removed ref to `.env`